application/actions/post_fixtures_test.go

```diff
@@ -160,6 +160,58 @@ func createFixturesForPostQuery(as *ActionSuite) PostQueryFixtures {
 	}
 }
 
+func createFixturesForSearchRequestsQuery(as *ActionSuite) PostQueryFixtures {
+	org := models.Organization{UUID: domain.GetUUID(), AuthConfig: "{}"}
+	createFixture(as, &org)
+
+	unique := org.UUID.String()
+	users := make(models.Users, 2)
+	userOrgs := make(models.UserOrganizations, len(users))
+	accessTokenFixtures := make([]models.UserAccessToken, len(users))
+	for i := range users {
+		users[i].UUID = domain.GetUUID()
+		users[i].Email = unique + "_user" + strconv.Itoa(i) + "@example.com"
+		users[i].Nickname = unique + "_auth_user" + strconv.Itoa(i)
+		users[i].AuthPhotoURL = nulls.NewString(users[i].Nickname + ".gif")
+		createFixture(as, &users[i])
+
+		userOrgs[i].UserID = users[i].ID
+		userOrgs[i].OrganizationID = org.ID
+		userOrgs[i].AuthID = unique + "_auth_user" + strconv.Itoa(i)
+		userOrgs[i].AuthEmail = unique + users[i].Email
+		createFixture(as, &userOrgs[i])
+
+		accessTokenFixtures[i].UserID = users[i].ID
+		accessTokenFixtures[i].UserOrganizationID = userOrgs[i].ID
+		accessTokenFixtures[i].AccessToken = models.HashClientIdAccessToken(users[i].Nickname)
+		accessTokenFixtures[i].ExpiresAt = time.Now().Add(time.Minute * 60)
+		createFixture(as, &accessTokenFixtures[i])
+	}
```

Comment:
This can be replaced with `test.CreateUserFixtures(as.DB, as.T(), 2)` eventually.